src/book/03-guides/01-main-codebase/03-camera-calibration.mdx

```diff
@@ -108,7 +87,7 @@ The calibration requires two steps, creating the `nbs` recording and using it to
 
 3.  Verify these values by running vision and checking the output in NUsight.
 
-    1. Ensure `NUsight.yaml` has enabled compressed images and vision objects.
+    1. Ensure `NetworkForwarder.yaml` has enabled compressed images and vision objects.
```

Comment:
Would also be worth adding which specific vision objects from `message.vision.*` we want them to add, and put those in parenthesis.

```suggestion
    1. Ensure `NetworkForwarder.yaml` has enabled compressed images (`message.output.CompressedImage`) and vision objects.
```

Reply:
Also not entirely relevant for this PR but sure :P